contracts/FlowIDTableStaking.cdc

```diff
@@ -733,35 +735,41 @@ pub contract FlowIDTableStaking {
                 }
             }
 
+            var totalRewardScale: UFix64 = 0.0
+
+            if totalStaked >= UFix64(100000000.0) {
+                // Maximum UFix64 divisor is 100M so we need to scale the numbers
+                // in order to not cause overflow
+                let div1000dividend = FlowIDTableStaking.epochTokenPayout / 1000.0
+                let div1000divisor = totalStaked / 1000.0
+                totalRewardScale = div1000dividend / div1000divisor
+            } else if totalStaked != 0.0 {
+                totalRewardScale = FlowIDTableStaking.epochTokenPayout / totalStaked
+            }
+
```

Comment:
This can still lose us accuracy in the fractional part of the number when scaling but it is 1000 times less likely to overflow. 👍